.github/workflows/build-and-install.yml

```diff
@@ -121,18 +125,19 @@ jobs:
           docker build . -f .github/dockerfiles/Dockerfile.build_test -t test --build-arg BASE=${{ matrix.distro }}
       - name: Regular build on ${{ matrix.distro }}
         run: |
-          docker run -w /netdata test /bin/sh -c 'autoreconf -ivf && ./configure && make -j2'
+          docker run --security-opt seccomp=unconfined -w /netdata test /bin/sh -c 'autoreconf -ivf && ./configure && make -j2'
```

Comment:
Why is this needed?

Reply:
Because of the runc developers can’t wrap their heads around returning `-ENOSYS` for systemcalls that are not present on the system which they do not know about instead of `-EPERM`, which violates normal Linux kernel API guarantees and in turn breaks fallback handling for newer system calls in libc.

See https://bugs.launchpad.net/cloud-images/+bug/1943049 for more details on this specific case, but this is not an isolated case (the same general type of issue also means that, for example, Alpine 3.13 cannot run in a container on older versions of Docker without this option).

Reply:
exacly